api/src/api/services/transactions/stellar/offrampTransaction.ts

```diff
@@ -7,14 +7,23 @@ import Big from 'big.js';
 const FUNDING_PUBLIC_KEY = FUNDING_SECRET ? Keypair.fromSecret(FUNDING_SECRET).publicKey() : '';
 const NETWORK_PASSPHRASE = Networks.PUBLIC;
 const MAX_TIME = Date.now() + 1000 * 60 * 10;
+const SEQUENCE_SHIFT_IN_SECONDS = 240; // 4 minutes
```

Comment:
I don't understand the name. What's a sequence shift? 😅

Reply:
Oh, since the sequence seems to be the ledger shifted 32 bits... I just named it sequence shift (in time!). Is it confusing haha? We can change it of course. `SEQUENCE_TIME_SHIFT_SECONDS` ??

Reply:
But just from the name I wouldn't even know how to use it. How about we use different constants. 

```suggestion
const STELLAR_LEDGER_CLOSE_TIME_SECONDS = 6; 
const STELLAR_LEDGER_BUFFER_WINDOW_SECONDS = 5 * 60; // 5 minutes
/// This is used to add a buffer on top of our current ledger sequence number. We assume
const STELLAR_LEDGERS_PER_BUFFER = Math.ceil(STELLAR_LEDGER_BUFFER_WINDOW_SECONDS / STELLAR_LEDGER_CLOSE_TIME_SECONDS);
```

Reply:
Sorry I'm just seeing this. I changed it to `SEQUENCE_TIME_WINDOW_IN_SECONDS` and added a few comments. It would be comparable to your `STELLAR_LEDGER_BUFFER_WINDOW_SECONDS`. Added a comment on constants also.